.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
+
+on:
+  create:
+  push:
+    branches: master
+    paths:
+      - font-patcher
+      - src/glyphs/**
```

Comment:
If someone pushes a fixed `svg` to `src/svgs` the `packsvgs.yml` will run and create a new `original-source.ttf` and push it. But I guess that will NOT trigger this workflow unless one adds something somewhere. I have no clue what and where. Just that there are some ... hurdles if workflow commits shall trigger workflows :thinking: 

It is ok how it is now, but maybe you know the answer :-)

Reply:
Also after a release.yml run, I guess the zip inside the repo will not be updated although the version in the script has been changed (by the release workflow) :thinking:

Reply:
> If someone pushes a fixed `svg` to `src/svgs` the `packsvgs.yml` will run and create a new `original-source.ttf` and push it. But I guess that will NOT trigger this workflow unless one adds something somewhere. I have no clue what and where. Just that there are some ... hurdles if workflow commits shall trigger workflows 🤔
> 
> It is ok how it is now, but maybe you know the answer :-)

https://github.com/b-/Iosevka/blob/cc3babd8033f64086341820cfa014d026a14021f/.github/workflows/build.yml#L45

I don’t remember if this step actually functioned or not — I might have been missing something when I set it up. But the idea here is to set an API PAT (Personal Auth. Token) for the repository you want to trigger workflows _on_, add it as a secret to the repo you want to trigger workflows _from_, and then just call the second workflow from the first. 

I’ve done some research, and I don’t think it’s possible to trigger one workflow from another without a custom PAT. Actions Workflows get a token by default anyway, but it specifically has the ability to trigger other workflows disabled — that’s the issue there